File: Bookcase core/src/main/java/cz/vhromada/book/repository/BookRepository.java
package cz.vhromada.book.repository;

import cz.vhromada.book.domain.Book;

import org.springframework.data.jpa.repository.JpaRepository;

/**
 * An interface represents repository for books.
 *
 * @author Vladimir Hromada
 */
public interface BookRepository extends JpaRepository<Book, Integer> {
}

File: Bookcase core/src/main/kotlin/cz/vhromada/book/repository/BookRepository.kt
package cz.vhromada.book.repository

import cz.vhromada.book.domain.Book
import org.springframework.data.jpa.repository.JpaRepository

/**
 * An interface represents repository for book.
 *
 * @author Vladimir Hromada
 */
interface BookRepository : JpaRepository<Book, Int>
